kubernetes/kaniko/mvnKaniko/kaniko-template.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kaniko-image-creator
+spec:
+  containers:
+  - name: kaniko-image-creator
+    image: gcr.io/kaniko-project/executor:debug
+    args: [ 
+            "--context=git://github.com/giros-dit/semantic-data-aggregator.git#refs/heads/feature-105",
+#            "--context-sub-path=semantic-metrics-datamodels/",
+            "--dockerfile=kubernetes/kaniko/mvnKaniko/Dockerfile",
+            "--destination=192.168.159.9:31320/netflow2cds:latest",
```

Comment:
Whose IP address is this? Please use service names to avoid setting IP addresses manually.